core/src/main/java/com/graphhopper/routing/util/MotorcycleFlagEncoder.java

```diff
@@ -208,10 +207,8 @@ public IntsRef handleWayTags(IntsRef edgeFlags, ReaderWay way, EncodingManager.A
             setSpeed(true, edgeFlags, ferrySpeed);
         }
 
-        priorityWayEncoder.setDecimal(false, edgeFlags, PriorityCode.getFactor(handlePriority(way)));
-
-        // Set the curvature to the Maximum
-        curvatureEncoder.setDecimal(false, edgeFlags, PriorityCode.getFactor(10));
+        priorityWayEncoder.setDecimal(false, edgeFlags, PriorityCode.getValue(handlePriority(way)));
+        curvatureEncoder.setDecimal(false, edgeFlags, 10.0 / 7.0);
```

Comment:
Previously we had `PriorityCode.getFactor(10)=10/10.0=1`, but now we have `10.0/7.0`=1.43...`? Should this not be `1.0+0.5=1.5` or `PriorityCode.getValue(BEST)`?

Reply:
Sorry, my bad. You are right it was `10.0/7.0` also before and makes sense not to use `PriorityCode` for curvature.